adminSiteClient/VariablesAnnotationPage.tsx

```diff
@@ -711,14 +743,13 @@ class VariablesAnnotationComponent extends React.Component {
                 visible: false,
                 description: item.description,
             }))
-        const readonlyReorderItems: ColumnReorderItem[] =
-            VariablesAnnotationComponent.readOnlyColumnNamesFields.map(
-                ([label, id]) => ({
-                    key: id,
-                    visible: false,
-                    description: label,
-                })
-            )
+        const readonlyReorderItems: ColumnReorderItem[] = [
```

Comment:
Just flagging this here, and it's not new, but I have absolutely no clue why this all is called `reorder`. What are we reordering in the first place?

Reply:
I renamed this to ColumnInformation - the point of this data structure is to be kept in an array that signifies the user specified order, which is why I called it ColumnReorderItem. But ColumnInformation is prbably the better name for now. If you have an even better one let me know :)